sklearn/preprocessing/_data.py

```diff
@@ -381,14 +384,18 @@ class MinMaxScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
 
     _parameter_constraints: dict = {
         "feature_range": [tuple],
+        "output_dtype": [tuple, type],
         "copy": ["boolean"],
         "clip": ["boolean"],
     }
 
-    def __init__(self, feature_range=(0, 1), *, copy=True, clip=False):
+    def __init__(
+        self, feature_range=(0, 1), output_dtype=FLOAT_DTYPES, *, copy=True, clip=False
```

Comment:
```suggestion
        self, feature_range=(0, 1), *, copy=True, clip=False, output_dtype=FLOAT_DTYPES,
```

I would have placed the new argument after the `*` to make it keyword only. Keyword only arguments are easier to deal with when it comes to deprecations and it is easier to read for something like `MinMaxScaler` that has many arguments.

Reply:
Agreed, I'll make the change. Thanks!